src/option.rs

```diff
@@ -166,7 +166,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "\n\texpected: option[none]\n\t but was: option<\"Hello\"")]
+    #[should_panic(expected = "\n\texpected: option[none]\n\t but was: option<\"Hello\">")]
```

Comment:
Not like this was "wrong" but seems more expected to include the closing `>` in the expected panic message